src/shared/lib/OncoKbUtils.ts

```diff
@@ -19,8 +20,9 @@ const ONCOGENIC_CLASS_NAMES:{[oncogenic:string]: string} = {
 // (used for sorting purposes)
 const ONCOGENIC_SCORE:{[oncogenic:string]: number} = {
```

Comment:
I see.  Look at documentation and it seems TS enums don't work with dynamic accessors.  So you are doing the right thing.